cmd/bucket-replication.go

```
@@ -2958,16 +2967,20 @@ func (p *ReplicationPool) persistMRF() {
 				entries[e.versionID] = e
 			}
 		}
+		// queue all entries for healing before overwriting the node mrf file
+		if err := p.queueMRFHeal(); err != nil && !osIsNotExist(err) {
+			logger.LogIf(p.ctx, err)
```

should we be logging this error at all?

---

we can ignore errors on this